packages/odyssey-react-mui/src/theme/palette.ts

```diff
@@ -16,76 +16,76 @@ import * as Tokens from "@okta/odyssey-design-tokens";
 export const palette: ThemeOptions["palette"] = {
   mode: "light",
   common: {
-    black: Tokens.ColorNeutralDark,
-    white: Tokens.ColorPaletteNeutralWhite,
+    black: Tokens.HueNeutral900,
+    white: "#ffffff",
```

Comment:
We aligned on this last week and this works fine for me, but just flagging that we do have absolute white in neutrals now. So if it's clearner to call our HueNeutralWhite that's also fine

Reply:
Updatin'